sdk/examples/example.ts

```diff
@@ -13,7 +13,7 @@ import { IotaNamesTransaction } from '../src/iota-names-transaction.js';
 (async () => {
     const domain = `cool-domain-${Math.floor(Math.random() * 10000000)}.iota`;
 
-    const network = 'devnet';
+    const network = 'localnet';
```

Comment:
Let's revert it or maybe just `process.argv` and default to `default`
```suggestion
    const network = 'devnet';
```